src/components/Dialogs/Conditions/ConditionsManager.js

```diff
@@ -52,6 +53,7 @@ const DIMENSION_TYPE_TRUE_ONLY = 'TRUE_ONLY'
 const DIMENSION_TYPE_DATE = 'DATE'
 const DIMENSION_TYPE_TIME = 'TIME'
 const DIMENSION_TYPE_DATETIME = 'DATETIME'
+const DIMENSION_TYPE_ORGANISATION_UNIT = 'ORGANISATION_UNIT'
```

Comment:
In dhis2 terminology these are "value type", but probably makes sense from an ER pov.